e2e/test/scenarios/sharing/public-sharing-embed-button-behavior.cy.spec.js

```diff
@@ -279,6 +279,54 @@ describe("#39152 sharing an unsaved question", () => {
   });
 });
 
+describe("metabase#46893 - popover should appear next to button, not at the top left of the page", () => {
```

Comment:
these two tests check if the menu's and button's bounding rectangles are near each other - it doesn't have to be exact but just a few pixels considering offsets and paddings. as long as the popover isn't appearing at (x = 0, y = 0), then we're ok